crt_portal/static/js/complaint_view_filters.js

```diff
@@ -103,6 +103,7 @@
    * @returns {String} Param in the format of {key}={value}
    */
   function makeQueryParam(key, value) {
+    gtag('event', 'search_filter', { filter: key });
```

Comment:
is this going to make separate events for each param? if so, is that what we want?

This is kind of open for discussion I guess, but if you want to see what combinations are being used, maybe:

`[...Object.keys(params)].sort().join(' ')` to get a list of keys?